### ISSUE TITLE
Auto v7 failing under unknown circumstances (maybe npx?)

### ISSUE THREAD

**Issue by zephraph** [intuit/auto#442]
**Describe the bug**

I don't have a clear understanding of why this failure is happening, but I _suspect_ it's still an npx issue. 

I don't have a lot of time to dig into it right now, so I've just manually downgraded our version. I'll poke at it this evening though. 

Here's one of our [recent auto failures](https://circleci.com/gh/artsy/renovate-config/3235?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). 

```
npx: installed 149 in 10.052s
TypeError: plugin is not a constructor
    at Object.loadPlugin [as default] (/home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/utils/load-plugins.js:51:12)
    at /home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js:1013:67
    at Array.map (<anonymous>)
    at Auto.loadPlugins (/home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js:1013:14)
    at Auto.<anonymous> (/home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js:166:30)
    at step (/home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js:43:23)
    at Object.next (/home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js:24:53)
    at fulfilled (/home/circleci/.npm/_npx/113/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js:15:58)
Exited with code 1
```

**Comment by adierkens** [comment]
:rocket: Issue was released in v7.0.3 :rocket:

**Comment by zephraph** [comment]
I'm not 100% sure that this is fixed. 

I've got another [failure](https://circleci.com/gh/artsy/renovate-config/3305) that should be running the latest version. I'm going to verify though, give me a min.

**Comment by zephraph** [comment]
I can confirm it's still failing in `v7.0.4`

**Comment by hipstersmoothie** [comment]
Damn

**Comment by zephraph** [comment]
Maybe I should just drop `npx` and do something else.

**Comment by hipstersmoothie** [comment]
I think it’s good that you find these bugs . It’s a valid use case. It will
be a bug in the load-plugins file

**Comment by zephraph** [comment]
This is fixed by #480
